Demo/Common/Networking/PaymentsRequest.swift

```diff
@@ -29,7 +29,7 @@ internal struct PaymentsRequest: APIRequest {
         var container = encoder.container(keyedBy: CodingKeys.self)
 
         let currentConfiguration = ConfigurationConstants.current
-        let amount = data.order?.remainingAmount ?? currentConfiguration.amount
+        let amount = data.amount ?? currentConfiguration.amount
```

Comment:
maybe don't let the merchant worry about that and just provide the right amount as discussed offline

Reply:
hm.. in this case this is not a mobile integration, but a "backend code" 🤔
Not sure how can we do this..

Reply:
don't think I understand what you're saying! we discussed this before, right?